### PR TITLE
Created Projects & Experience Page, and added content

### PR DESCRIPTION
**Pull request by jessdejong**
Added files and images that are relevant to my projects, and added some descriptions of a couple projects and my past experiences.

### CODE REVIEW

**Comment by jessdejong** [comment]
It still uses the strong tags here, but in another branch I have them fixed... I was thinking that if I left them in here it might end up being easier to merge, but if you think I should change it here as well I'll go ahead and do that.

File: portfolio/src/main/webapp/projects_experience.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <div id="content">
+      <h1>Projects</h1>
+      <h2>Spherical Inverter</h2>
+      <div class="speech-bubble">
+        <img src="images/Spherical_Inverter_Screenshot.png" alt="Spherical Inverter Screenshot" width="400" border="1px solid black">
+        <p>
+          A MATLAB-based GUI application that visualizes three-dimensional spherical inversion. 
+          Performs inversion on spheres, cylinders, ellipsoids, custom point sets, and MATLAB 
+          functions passed by the user. 
+          <a href="files/Spherical_Inverter_Research_Report.pdf" target="_blank">Resgarch Report</a>.
```
Comment (estep-google) [inline portfolio/src/main/webapp/projects_experience.html:31]: Do you intend for this to be on a new line or a new sentence that directly follow the previous content?

File: portfolio/src/main/webapp/projects_experience.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <meta charset="UTF-8">
+    <title>Jess DeJong - STEP Intern</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="header">
+      <img src="images/Profile_photo_Jess.jpg" alt="Profile photo of Jess" height="250">
+      <h1>Jess's Portfolio</h1>
+      <strong>
```
Comment (estep-google) [inline portfolio/src/main/webapp/projects_experience.html:13]: I'm going to hold approval until the other PR is merged, just for clarity here.

File: portfolio/src/main/webapp/projects_experience.html (hunk 1 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/projects_experience.html:31]: New sentence that directly follows the previous line content

Comment (jessdejong) [reply portfolio/src/main/webapp/projects_experience.html:31]: Should I put that link on the previous line, and then wrap that line from there?

**Comment by jessdejong** [comment]
Realized that this is a different html file (projects_experience.html) and not index.html, which was updated in master. Added some additional commits to format the speech bubbles as cards for the projects page, and resolved the header issues.

File: portfolio/src/main/webapp/projects_experience.html (hunk 2 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/projects_experience.html]: Realized that this header is for projects_experience.html, and not index.html. Had to manually update this header to be correct.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -107,3 +107,11 @@ body {
...
   color: orange;
 }
+
+/* Styling layout cards */
+.card {
+  margin: 20px;
+  margin-left: 0px;
+  position: relative;
+  padding-right: 20px;
+}
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:117]: Please put this class by the `.speech-bubble` class. They go together, so they should be defined by each other.

**Comment by jessdejong** [comment]
I put the related css declarations near one another, as well as alphabetized the css properties that I hadn't done before.

**Pull request merged**